tests/hooks/test_firebolt.py

```diff
@@ -237,3 +235,26 @@ def test_engine_action_start_default(self, conn_params_call):
 
         with self.assertRaises(FireboltError):
             self.db_hook.engine_action(None, "start")
+
+    def test_run_returns_results(self):
+        sql = ["SQL1", "SQL2"]
+        self.cursor.fetchall.return_value = [(1, 2)]
+        res = self.db_hook.run(sql, handler=lambda cur: cur.fetchall())
+        assert res == [[(1, 2)], [(1, 2)]]
+
+        sql = "SQL1; SQL2"
+        res = self.db_hook.run(
+            sql,
+            handler=lambda cur: cur.fetchall(),
+            return_last=False,
+            split_statements=True,
+        )
+        assert res == [[(1, 2)], [(1, 2)]]
+
+        res = self.db_hook.run(
+            sql,
+            handler=lambda cur: cur.fetchall(),
+            return_last=True,
+            split_statements=True,
```

Comment:
Let's use airflow's own [handlers](https://github.com/apache/airflow/blob/1c98e51f12d2548b8d657540c49b685fb19e8e65/providers/src/airflow/providers/common/sql/hooks/sql.py#L75-L88). They do the same thing (essentially), but also let us test their compatibility with us.
I think this is would constitute a common use case.

Reply:
Better [link](https://github.com/apache/airflow/blob/main/providers/src/airflow/providers/common/sql/hooks/handlers.py#L52-L75) - just saw the above is deprecated.

Reply:
Oh nice, thanks for pointing